src/app/zap-templates/zcl/data-model/chip/thermostat-cluster.xml

```diff
@@ -189,21 +207,21 @@ limitations under the License.
       <description>
         The Current Weekly Schedule Command is sent from the server in response to the Get Weekly Schedule Command.
       </description>
-      <arg name="numberOfTransitionsForSequence" type="ENUM8"/>
-      <arg name="dayOfWeekForSequence" type="DayOfWeek"/>
-      <arg name="modeForSequence" type="ModeForSequence"/>
-      <arg name="payload" type="INT8U" array="true"/>
+      <arg name="NumberOfTransitionsForSequence" type="INT8U"/>
+      <arg name="DayOfWeekForSequence" type="DayOfWeek"/>
+      <arg name="ModeForSequence" type="ModeForSequence"/>
+      <arg name="Tayload" type="ThermostatScheduleTransition" array="true"/>
```

Comment:
Nit, is this supposed to be payload or transitions?

Reply:
"Transitions".  Fixed.